vm/vm.go

```diff
@@ -369,6 +369,7 @@ func (vm *VM) Initialize(
 		vm.Logger(),
 		"",
 		syncRegistry,
+		nil,
```

Comment:
Can you add a TODO to populate this?

Reply:
this is minVersion, I don't think it needs to be populated.